src/features/search-integration.md

```diff
@@ -0,0 +1,38 @@
+# Search Integration
+
+The VAT system provides a search integration with the [GFBio search](https://search.gfbio.org/) that allows users to transfer search results directly to the VAT system.
+
+## Searching for data
+
+![Visualizable in VAT](media/search-vis.png)
+
+To search for data, users can enter a search term in the search bar and press the `Enter` key. This will show the search results.
+Users can filter the search results by selecting the facet _Visualizable in VAT_.
```

Comment:
Maybe facet is not something a user understands. Better say that "Visualizable in VAT" can be selected on the left side?

Reply:
[dad8baf](https://github.com/umr-dbs/vat-docs/pull/12/commits/dad8baf69ad5be19dad55bbb0c63982e3f23b539)